### DISCUSSION TITLE
Failed to create the chia daemon after keys regen (Solved)

### DISCUSSION THREAD

**Discussion by drwolf00** [Chia-Network/chia-blockchain#6812] [Support]
I want to connect harvester on linux to main PC.
After i regenerate keys using chia init -c [directory] as described in [wiki](https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines), daemon not start.

**Comment by drwolf00** [comment]
I am not stop daemon before regenerate keys.
On other PC i stop daemon and all work as in wiki

**Comment by BrandtH22** [comment]
@drwolf00 , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
